README.md

```diff
@@ -86,10 +87,15 @@ jobs:
           repository: myorg/myimage
           tag: ${{ steps.docker-build.outputs.tag }}
           # fail_threshold: medium
+          # ignore_list: |
+          #   CVE-2020-1234
+          #   CVE-2020-5678
       # Access scan results in later steps
       - run: echo "${{ steps.docker-scan.outputs.total }} total vulnerabilities."
 ```
 
+>:warning: **Note**: The `ignore_list` can either be a multi-line string (like the example) or a comma-separated list of CVE IDs.
```

Comment:
I didn't know about this markdown callout feature! Nifty 😎 But I think this should be part of the parameter documentation on line 24.

Reply:
It's just quoted text with an emoji :)

>ℹ️ Ta-dah!